agent/src/test/java/org/jfrog/teamcity/agent/gradle/GradleBuildInfoAgentListenerTest.java

```diff
@@ -174,10 +174,14 @@ public void testBeforeRunnerStart() throws Exception {
         AgentRunningBuild agentRunningBuild = EasyMock.createMock(AgentRunningBuild.class);
         EasyMock.expect(agentRunningBuild.getBuildLogger())
                 .andReturn(EasyMock.createMock(BuildProgressLogger.class)).anyTimes();
+
         File tempDir = Files.createTempDir();
+        File buildTempDir = Files.createTempDir();
```

Comment:
It looks like ```buildTempDir``` is redundant. ```tempDir``` can be used instead.

Reply:
I added this, cause this reflects the change.
The proposed implementation uses 2 different directories, for different purposes.

As such I reflected this change in the Unit-Test, since this is accurate.
If I should change this to use 1 directory, instead of 2 then I will, but that would go against the purpose of the Unit-Test wouldn't it?

Greets Orion.

Reply:
Yes - you're right.